### PR TITLE
SB-30518 - Addition of session id

### PR DESCRIPTION
**Pull request by surabhi-mahawar**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-30518" title="SB-30518" target="_blank">TaskSB-30518</a>  Addition of Session Id
  </td></table>
  <br />

Added session id in UCI Response exhaust

### CODE REVIEW

**Comment by surabhi-mahawar** [comment]
@manjudr 
Can you please review and merge this.

File: data-products/src/main/scala/org/sunbird/analytics/exhaust/uci/UCIResponseExhaustJob.scala [hunk 1]
```diff
@@ -19,12 +19,12 @@ object UCIResponseExhaustJob extends optional.Application with BaseUCIExhaustJob
...
-    "Question Title", "Question Description", "Question Duration", "Question Score", "Question Max Score",
+  private val columnsOrder = List("Message ID", "Conversation ID", "Conversation Name", "Device ID", "Session Id", "Question Id", "Question Type",
+    "Question Title", "Question Description", "Question Response Time", "Question Score", "Question Max Score",
     "Question Options", "Question Response", "X Path", "Eof", "Timestamp")
   private val columnMapping = Map("mid" -> "Message ID", "conversation_id" -> "Conversation ID", "conversation_name" -> "Conversation Name",
-    "device_id" -> "Device ID", "question_id" -> "Question Id", "question_type" -> "Question Type", "question_title" -> "Question Title",
-    "question_description" -> "Question Description", "question_duration" -> "Question Duration", "question_score" -> "Question Score", "question_maxscore" -> "Question Max Score",
+    "device_id" -> "Device ID", "session_id" -> "Session Id", "question_id" -> "Question Id", "question_type" -> "Question Type", "question_title" -> "Question Title",
```
Comment (manjudr) [inline data-products/src/main/scala/org/sunbird/analytics/exhaust/uci/UCIResponseExhaustJob.scala:26]: Can we have a test case to check if the property exists in the report?

**Review by manjudr — COMMENTED** [review]
Please raise this PR to the release-4.10.5 branch..
And we have a production deployment the next day, although we are delivering these updates today?

**Comment by surabhi-mahawar** [comment]
@manjudr 
If you have a production deployment, then please merge this later, as this is not a part of the deployment plan for us. We have not yet updated the corresponding code for this on any of the testing environments.

**Comment by manjudr** [comment]
Please raise the changes to the release-5.0.0 branch.

**Comment by surabhi-mahawar** [comment]
Moved to other [PR](https://github.com/Sunbird-Ed/sunbird-data-products/pull/557)

**Pull request closed without merging**